src/graphql/data/nft/Collection.ts

```diff
@@ -1,12 +1,12 @@
-import graphql from 'babel-plugin-relay/macro'
+import { gql } from '@apollo/client'
```

Comment:
we haven't done it yet but fyi, to reach parity with relay i think you'll have to use the macro https://www.apollographql.com/docs/react/performance/babel/

Reply:
Thank you for this, significantly improved load time. Will make the change with the migration.